### DISCUSSION TITLE
How can I make the badge round on Daisy 2.52?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1990] [Q&A] [answered]
Hi

https://play.tailwindcss.com/mYd3BWmQ6t

I have tied adding the btn-circle class in it and it does what I need, but its bigger?

Is there a class that I use or adding a style could just do the trick?

Thank you so much?

**Comment by saadeghi** [comment, marked as answer]
It is circle by default but it also stretch based on the text width.  
And it's not really a good idea to make it circle unless you're sure you're no putting more than a limited amount of character in it (imagine the text is `999999`, how big is the circle going to be...)  

You can add `aspect-square` Tailwind CSS class to any element to make sure it has equal width and height
```html
<div class="indicator">
  <span class="indicator-item badge aspect-square badge-secondary">9</span> 
  <button class="btn">inbox</button>
</div>
```
You can also add any width or height class names to change the size based on your content
```html
<div class="indicator">
  <span class="indicator-item badge h-10 w-10 badge-secondary">9999</span> 
  <button class="btn">inbox</button>
</div>
```

See examples: https://play.tailwindcss.com/spKTLkz8Yt
